themes/sgsc/sass/_about-partners.scss

```diff
@@ -153,6 +155,9 @@
     .entry-header {
         max-width: 1024px;
         height: 375px; 
+      }
```

Comment:
Need to change the height of `entry-title` to 10 rem to be consistent with the other pages at Desktop width

Reply:
Header fixed.